packages/components/src/Banner/Banner.css

```diff
@@ -10,7 +10,7 @@
   padding: var(--flash--inset);
   color: var(--color-black);
   background-color: var(--color-grey--lightest);
-  align-items: flex-start;
+  align-items: center;
```

Comment:
@rodrigoeidelvein this breaks the alignment from what we expect / what was previously there

You can see this pushes the dismiss button to the center of the banner:
![image](https://user-images.githubusercontent.com/39704901/213810991-74894bfc-b383-4cfd-a2fc-f2533e60105a.png)

It should remain aligned to the top:
![image](https://user-images.githubusercontent.com/39704901/213811082-b2e6d17c-10b4-4831-b721-44fe8073c5f4.png)